sklearn/neighbors/tests/test_nearest_centroid.py

```diff
@@ -3,17 +3,30 @@
 """
 import numpy as np
 import pytest
-from numpy.testing import assert_array_equal
+from numpy.testing import assert_array_almost_equal, assert_array_equal
+from scipy import sparse as sp
 
 from sklearn import datasets
 from sklearn.neighbors import NearestCentroid
-from sklearn.utils.fixes import CSR_CONTAINERS
```

Comment:
I think that you mess up when solving some conflicts with `main`.

We need to test with parametrization and with all the different `CSR_CONTAINERS`

Reply:
Coul you revert those changes.